backend/router/routes.js

```diff
@@ -1,7 +1,9 @@
 const express = require('express');
+const bodyParser = require('body-parser');
 
 const router = express.Router(); // get an instance of the express Router
-
+// parse various different custom JSON types as JSON
+router.use(bodyParser.json());
```

Comment:
Why is this used here? .I do not think this is necessary as body parser is already used in the application context

Reply:
Fixed